flytekit/models/execution.py

```diff
@@ -349,6 +359,7 @@ def from_flyte_idl(cls, pb2_object):
             error=error,
             outputs=outputs,
             phase=pb2_object.phase,
+            started_at=pb2_object.started_at,
```

Comment:
Protobuf timestamps are a bit annoying in converting to/from python datetimes. See: 

https://github.com/lyft/flytekit/blob/d4cfedc4c580f08bf904e6e474a0b948a4608737/flytekit/models/literals.py#L136

https://github.com/lyft/flytekit/blob/d4cfedc4c580f08bf904e6e474a0b948a4608737/flytekit/models/literals.py#L152

Reply:
Updated, modeled after:
https://github.com/lyft/flytekit/blob/d4cfedc4c580f08bf904e6e474a0b948a4608737/flytekit/models/node_execution.py#L75
https://github.com/lyft/flytekit/blob/d4cfedc4c580f08bf904e6e474a0b948a4608737/flytekit/models/node_execution.py#L89